src/ez/impl/MySQLDB.java

```diff
@@ -89,6 +89,20 @@ public boolean hasColumn(String table, String column) {
         schema, table, column);
   }
 
+  @Override
+  public boolean hasForeignKey(String sourceTable, String sourceColumn, String foreignTable, String foreignColumn) {
+    return null != selectSingleRow("SELECT `COLUMN_NAME`"
+        + " FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE WHERE TABLE_SCHEMA = ? AND TABLE_NAME = ? AND COLUMN_NAME = ? AND REFERENCED_TABLE_NAME = ? AND REFERENCED_COLUMN_NAME = ? LIMIT 1",
```

Comment:
formatting

Reply:
not sure what is wrong with the formatting looks similar to hasColumn and I have the correct formatter on ecllipse for Ender. Is there something separate I have to go for EZDB?

Reply:
lines should not be longer than 120 characters

Reply:
fixed